### PR TITLE
Blp 69 column rename

### PR DESCRIPTION
**Pull request by ChiuMungZitAlexander**

### CODE REVIEW

File: src/client/components/boardColumn/index.js [hunk 1]
```diff
@@ -54,6 +60,40 @@ class List extends React.Component {
...
+      allowDrag: this.state.isColumnTitleEditing,
+    });
+  }
+
+  generateColumnTitle() {
+    if (this.state.isColumnTitleEditing) {
+      return (
+        <section className="column-title-editing">
+          <input type="text"
+            ref={(c) => {
```
Comment (beltekylevi) [inline src/client/components/boardColumn/index.js:77]: what is the purpose of this?

File: src/client/components/boardColumn/index.js [hunk 2]
```diff
@@ -187,22 +243,28 @@ class List extends React.Component {
         ref={(elem) => {
           this.wrapper = elem;
         }}>
-        <div className="board-column" draggable="true"
+        <div className="board-column"
+          draggable={this.state.allowDrag ? 'true' : 'false'}
+          ref={(elem) => {
```
Comment (beltekylevi) [inline src/client/components/boardColumn/index.js]: what is the purpose of this?

File: src/client/components/boardColumn/index.js [hunk 3]
```diff
@@ -187,22 +243,28 @@ class List extends React.Component {
...
+            ref={(elem) => {
+              this.editIcon = elem;
+            }}>
             </div>
             {this.state.isEditing ?
               <Menu
                 columnId={this.props.column._id}
                 closeDropDownMenu={this.closeDropDownMenu}
                 deleteColumn={this.props.deleteColumn}
+                editColumn={this.editColumn.bind(this)}
```
Comment (beltekylevi) [inline src/client/components/boardColumn/index.js]: bind `this` in the constructor

**Review by beltekylevi — CHANGES_REQUESTED** [review]

File: src/client/components/boardColumn/index.js (hunk 1 above)
Comment (ChiuMungZitAlexander) [reply src/client/components/boardColumn/index.js:77]: to get the input so I can use this.input.focus() to automatically set focus to input after click edit

**Review by beltekylevi — APPROVED** [review]

**Pull request merged**